spec/02-integration/13-vaults/04-echo_spec.lua

```diff
@@ -0,0 +1,189 @@
+local helpers = require "spec.helpers"
+
+
+local ADMIN_HEADERS = {
+  ["Content-Type"] = "application/json",
+}
+
+
+for _, strategy in helpers.each_strategy() do
+  describe("Vault configuration #" .. strategy, function()
+    local admin_client
+    local proxy_client
+
+    lazy_setup(function()
+      local bp = helpers.get_db_utils(strategy, { "vaults" }, { "secret-response" }, { "echo" })
+
+      local route = bp.routes:insert {
+        paths = { "/" },
+      }
+
+      bp.plugins:insert {
+        name    = "secret-response",
+        route   = { id = route.id },
+      }
+
+      assert(helpers.start_kong {
+        database = strategy,
+        prefix = helpers.test_conf.prefix,
+        nginx_conf = "spec/fixtures/custom_nginx.template",
+        plugins = "secret-response",
+        vaults = "echo",
+      })
+    end)
+
+    before_each(function()
+      admin_client = assert(helpers.admin_client())
+      proxy_client = assert(helpers.proxy_client())
+    end)
+
+    after_each(function()
+      if admin_client then
+        admin_client:close()
+      end
+      if proxy_client then
+        proxy_client:close()
+      end
+    end)
+
+    lazy_teardown(function()
+      helpers.stop_kong()
+    end)
+
+    it("is not sticky", function()
```

Comment:
nit: I believe I understand what you mean by `is not sticky`, but I think this could use a more verbose/explicit description.